tests/test_core.py

```diff
@@ -0,0 +1,65 @@
+import os
```

Comment:
The files in directory `tests` don't have to follow the prefix "test_" in their name.

Reply:
Instead, something like `tests/core.py` and `ext-tangleid.py` makes sense.

Reply:
This looks better 👍